modes.xml

```diff
@@ -35,23 +35,31 @@
       <program name="lt-proc -w">
         <file name="spa.automorf.bin"/>
       </program>
-      <program name="cg-proc -w">
+      <program name="cg-proc">
         <file name="spa.rlx.bin"/>
       </program>
-      <program name="apertium-tagger -g">
+      <program name="apertium-tagger -g $2 -p">
         <file name="spa.prob"/>
       </program>
+      <program name="apertium-extract-caps"/>
```

Comment:
I don't think it's useful to extract caps in a pipeline that's never going to reinsert them